tests/components/homewizard/test_config_flow.py

```diff
@@ -75,9 +79,19 @@ async def test_discovery_flow_works(hass, aioclient_mock):
     with patch(
         "homeassistant.components.homewizard.async_setup_entry",
         return_value=True,
-    ):
+    ), patch("aiohwenergy.HomeWizardEnergy", return_value=get_mock_device()):
+        result = await hass.config_entries.flow.async_configure(
+            flow["flow_id"], user_input=None
+        )
+    assert result["type"] == RESULT_TYPE_FORM
+    assert result["step_id"] == "discovery_confirm"
+
+    with patch(
+        "homeassistant.components.homewizard.async_setup_entry",
+        return_value=True,
```

Comment:
Not sure what this is for, surely the return value should already be true if the other patching is correct?

Reply:
Because of this: https://github.com/home-assistant/core/pull/55812#discussion_r781775964

> **MartinHjelmare**
> We normally patch the integration setup function on all create entry results to avoid setting up the whole integration. See the scaffold script test template for example.